src/intersections/triangles.jl

```diff
@@ -36,7 +36,7 @@ function intersecttype(f::Function, s::Segment{3,T}, t::Triangle{3,T}) where {T}
 
     W₂ = A × D
     tᵥ = W₂ ⋅ C
-    
```

Comment:
Please disable whitespace changes in your editor in future PRs so that we can concentrate on the actual meaningful changes. 👍🏽 🙂

Reply:
OK. Sorry for that.